src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (invertedRobot.hasOwnProperty(robot[key])) {
```

Comment:
The use of 'hasOwnProperty' is correct for checking if a property exists in an object. However, consider using 'Object.prototype.hasOwnProperty.call(invertedRobot, robot[key])' to avoid potential issues if the 'robot' object has its own 'hasOwnProperty' property.